features/ALS015_Alignment-representation-zero-length-final-segment.feature

```diff
@@ -0,0 +1,23 @@
+@ALS
+@version1
+Feature: ALS015 - Alignment Representation Zero-Length Final Segment
+The rule verifies that the alignment geometry (representation) curve
+ends with a discontinuous segment with length = 0 and Transition = "DISCONTINUOUS".
+
+Background: Validating final segment of alignment geometry (representation).
+  Given A model with Schema "IFC4.3"
+  Given An IfcAlignment
+  Given Its attribute Representation
+  Given Its attribute Representations
+  Given Its attribute Items
+  Given Its attribute Segments
+  Given Its final segment
+
+@E00020
+Scenario:
+  Then The SegmentLength of the final segment must be 0
```

Comment:
Would the following work? 
```
Given Its final segment 
Given its attribute SegmentLength or Length

Then The value must be equal to 0
```

Reply:
But maybe it's too simplistic due to the involvement of the geometric tolerance (which isn't the case with just plain integer to integer comparisons)

Reply:
I guess I could add the additional `Given` statement.  It would eliminate a few lines from the `then` step implementation.  Is that the preferred approach stylistically?